packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -78,9 +80,13 @@ export class GraphQLGenerator {
 
   public readModelResolverBuilder(
     readModelClass: AnyClass
-  ): GraphQLFieldResolver<any, GraphQLResolverContext, { filter: Record<string, ReadModelPropertyFilter> }> {
+  ): GraphQLFieldResolver<any, GraphQLResolverContext, ReadModelRequestArgs> {
     return (parent, args, context, info) => {
-      const readModelEnvelope = toReadModelRequestEnvelope(readModelClass.name, args, context)
+      let isPaginated = false
+      if (info.fieldName && info.fieldName === `List${pluralize(readModelClass.name)}`) {
```

Comment:
I think you can do just:
```suggestion
      if (info?.fieldName === `List${pluralize(readModelClass.name)}`) {
```